library/custom_template/personalize.php

```diff
@@ -288,161 +288,155 @@ function personalize_save() {
+                            $resTemplates = sqlStatement($sql, $arval1);
+                        }
+                        while ($rowTemplates = sqlFetchArray($resTemplates)) {
+                            $cntxt = '';
+                            if (!$filter_context ?? null) {
+                                $context = sqlQuery("SELECT * FROM customlists WHERE cl_list_slno=?", array($rowTemplates['cl_list_id']));
+                                $cntxt .= $context['cl_list_item_long'] . "->";
+                            }
+                            if (!$filter_users ?? null) {
+                                $context = sqlQuery("SELECT * FROM users WHERE id=?", array($rowTemplates['tu_user_id']));
+                                $cntxt .= $context['username'] . "->";
+                            }
+                            echo "<option value='" . attr($rowTemplates['cl_list_slno'] . "|" . $rowTemplates['tu_user_id']) . "'>" . text($cntxt . $rowTemplates['cl_list_item_long']) . "</option>";
+                        }
+                        $sqlorphan = "SELECT * FROM customlists WHERE cl_list_type=3 AND cl_deleted=0 AND cl_list_slno" .
+                            " NOT IN (SELECT DISTINCT tu_template_id FROM template_users) " . $where . " ORDER BY cl_list_id,cl_list_item_long";
+                        if (empty($where)) {
+                            $resorphan = sqlStatement($sqlorphan);
+                        }
+                        while ($roworphan = sqlFetchArray($resorphan)) {
+                            $cntxt = '';
+                            if (!$filter_context ?? null) {
+                                $context = sqlQuery("SELECT * FROM customlists WHERE cl_list_slno=?", array($roworphan['cl_list_id']));
+                                $cntxt .= $context['cl_list_item_long'] . "->";
+                            }
+                            echo "<option value='" . attr($roworphan['cl_list_slno'] . "|") . "'>" . text($cntxt . $roworphan['cl_list_item_long']) . "</option>";
+                        }
+                        ?>
+                    </select>
+                </div>
+                <div class="col-sm-2 text-center">
+                    <button name="remove" class="btn btn-secondary" onclick="jsub_selected(document.myform,'personalized','topersonalized')">&raquo;</button>
+                    <br />
+                    <button name="remove" class="btn btn-secondary" onclick="check_user_category(document.myform,'topersonalized','personalized')">&laquo;</button>
+                </div>
+                <div class="col-sm-5">
+                    <select multiple class="form-control" name="personalized[]" id="personalized" size="6">
+                        <?php
+                        $where = '';
+                        if ($filter_context ?? null) {
+                            $where .= " AND cl_list_id='" . $filter_context . "'";
```

Comment:
bookmarking this for issue

Reply:
Really should be bound but this is why I hate touching these old scripts, everything needs to be done to them....
Again i'd appreciate if @stephenwaite would fix these or it will have to wait for my next PR